src/v1/sources/shopify/webhookTransformations/serverSideUtlis.js

```diff
@@ -39,7 +39,25 @@ const createPropertiesForEcomEventFromWebhook = (message) => {
   return mappedPayload;
 };
 
+/**
+ * Returns the anonymousId from the noteAttributes array in the webhook event
+ * @param {Object} event
+ * @returns {String} anonymousId
+ */
+const getAnonymousIdFromAttributes = async (event) => {
+  let anonymousId = null;
+  if (isDefinedAndNotNull(event) && isDefinedAndNotNull(event.note_attributes)) {
```

Comment:
`const getAnonymousIdFromAttributes = async (event) => {
  if (!isDefinedAndNotNull(event) || !isDefinedAndNotNull(event.note_attributes)) {
    return null; // Return early if event or note_attributes is invalid
  }

  const noteAttributes = event.note_attributes;
  const rudderAnonymousIdObject = noteAttributes.find(
    (attr) => attr.name === 'rudderAnonymousId'
  );

  return rudderAnonymousIdObject ? rudderAnonymousIdObject.value : null;
};
`

Reply:
Try to avoid use `let` wherever possible - the code readability automatically improves